base/cvd/cuttlefish/host/commands/cvd/server_command/status_fetcher.cpp

```diff
@@ -241,10 +284,20 @@ Result<Json::Value> StatusFetcher::FetchGroupStatus(
                                  original_request.FileDescriptors()};
   auto [_, instances_json, group_response] =
       CF_EXPECT(FetchStatus(group_request));
-  CF_EXPECT(
-      group_response.status().code() == cvd::Status::OK,
-      fmt::format("Running `cvd status --all_instances` for group \"{}\" failed",
-                  group.GroupName()));
+  if (group_response.status().code() != cvd::Status::OK) {
```

Comment:
Kind of an ambiguous question, but should this case be represented by a failed `Result` rather than a successful `Result` with a non-`OK` status? What distinguishes these conditions?

Probably out of the scope of this change though.